tests/test_libec2uploadimg.py

```diff
@@ -0,0 +1,2102 @@
+#!/usr/bin/python3
+#
+# Copyright (c) 2015 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of ec2utils
+#
+# ec2utils is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version.
+#
+# ec2utils is distributed in the hope that it will
+# be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ec2utils. If not, see
+# <http://www.gnu.org/licenses/>.
+#
+
+# import datetime
+# import dateutil.relativedelta
+import logging
+import pytest
+# import time
```

Comment:
No commented imports, if we don't need them please remove.

Reply:
done!